### DISCUSSION TITLE
VcLayerImagery的子组件只有最后一个生效

### DISCUSSION THREAD

**Discussion by HHKMFZY** [zouyaoji/vue-cesium#643] [Q&A]
![image](https://github.com/user-attachments/assets/333a9a54-30f8-4df3-98be-305724ef274d)
如图，只有villboun:广东省图层生效，其他两个都没有加载

**Comment by zouyaoji** [comment]
VcLayerImagery和provider要配套使用，也就要多个VcLayerImagery哦。 可以 参考一下vue-cesium-demo。
